Tests/PostgresNIOTests/New/Messages/CloseTests.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  File.swift
+//  
+//
+//  Created by Fabian Fett on 12.01.21.
+//
+
+import XCTest
```

Comment:
Ditch Xcode template headers.
```suggestion
import XCTest
```